corehq/messaging/smsbackends/connectid/views.py

```diff
@@ -27,3 +33,55 @@ def receive_message(request, *args, **kwargs):
     )
     process_incoming(msg, phone_obj)
     return HttpResponse("")
+
+
+@csrf_exempt
+@connectid_token_auth
+def connectid_messaging_key(request, domain):
+    link = get_object_or_404(ConnectIDUserLink, commcare_user=request.user, domain=request.domain)
```

Comment:
I think you want to fetch by channel_id, which you should be able to get from the request, since you only have a user but no domain

Reply:
I also made this a POST request view to get channel_id from the request body. Fixed. [3cb58a1](https://github.com/dimagi/commcare-hq/pull/35273/commits/3cb58a13326649332d7ed85619397b072f55ae2c)